src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/nomis/GetAddressesForPersonTest.kt

```diff
@@ -38,7 +38,29 @@ class GetAddressesForPersonTest(
       """
           [
             {
-              "postalCode": "SA1 1DP"
+              "postalCode": "SA1 1DP",
+              "addressId": 123456,
+              "addressType": "someType",
+              "flat": "2",
+              "premise": "Cool Building",
+              "street": "BROOMFIELD ROAD",
+              "locality": "ERDINGTON",
+              "town": "Birmingham",
+              "county": "West Midlands",
+              "country": "England",
+              "primary": false,
+              "noFixedAddress": false,
+              "startDate": "2022-07-01",
+              "endDate": "2023-03-01",
+              "phones": [],
+              "addressUsages": [
+                {
+                  "addressId": 8681879,
+                  "addressUsage": "HOME",
+                  "addressUsageDescription": "Home",
+                  "activeFlag": false
+                }
+              ]
```

Comment:
Will this test work as we haven't defined anything for a person address yet? Or is it all based on the underlying model?